### PR TITLE
Added automatically open Android front camera support using option

### PR DESCRIPTION
**Pull request by AsminBudha**
- [x] Explain the **motivation** for making this change.
- [x] Provide a **test plan** demonstrating that the code is solid.
- [ ] Match the **code formatting** of the rest of the codebase.
- [ ] Target the `master` branch, NOT a "stable" branch.

## Motivation (required)
I was unable to open front camera by default when I add option cameraType: 'front' on Android.

What existing problem does the pull request solve?
Now, I can open the front camera by default if I add option cameraType: 'front' on Android too.
This functionality is working fine for Android version below  Android 6 Marshmallow.

## Test Plan (required)
Create an app that uses react-native-image-picker to open camera. Then add option cameraType: 'front' which opens front camera by default in android.

### CODE REVIEW

**Comment by HarshitMadhav** [comment]
Someone update on this pull request?

**Comment by wsliaw** [comment]
any update on this PR? this aligns the behavior of Android & iOS, thanks

**Comment by Peretz30** [comment]
Common guys. Merge please

**Comment by winardis** [comment]
hi are there any updates on this? 
Screen Shot 2019-11-27 at 10 33 48 AM

**Comment by santhoshvai** [comment]
Common guys. Merge please

File: android/src/main/java/com/imagepicker/ImagePickerModule.java [hunk 1]
```diff
@@ -80,6 +81,7 @@
   protected Uri cameraCaptureURI;
   private Boolean noData = false;
   private Boolean pickVideo = false;
+  private int frontCamera=Camera.CameraInfo.CAMERA_FACING_BACK;
```
Comment (mattijsf) [inline android/src/main/java/com/imagepicker/ImagePickerModule.java]: Would be better to name this `cameraFacing` as it can be either back or front.

**Comment by litinskii** [comment]
Common guys. Merge please

**Pull request closed without merging**